.github/workflows/test.yml

```diff
@@ -258,6 +263,40 @@ jobs:
           cd prusti-contracts/prusti-contracts-test/
           cargo +stable build
 
+  release-build:
```

Comment:
Important: the new `release-build` job has to be added to the `needs` key of the `can-merge` job. Without that, pull requests can be merged even if the `release-build` fails or is cancelled.